lib/v4/migration-helpers/update-plugin-folder-structure/utils/create-content-type-index.js

```diff
@@ -0,0 +1,26 @@
+const { join } = require('path');
+const fs = require('fs-extra');
+
+const { importFilesToIndex, addModulesToExport } = require('../transforms');
+
+const createDirectoryIndex = require('./create-directory-index');
+
+async function createContentTypeIndex(v4PluginPath, dir) {
+  const hasDir = await fs.pathExists(dir);
+  if (!hasDir) return;
+
+  const indexPath = join(dir, 'index.js');
+
+  await fs.copy(join(__dirname, '..', 'utils', 'module-exports.js'), indexPath);
+  const dirContent = await fs.readdir(dir, { withFileTypes: true });
+  const directoriesToImport = dirContent.filter((fd) => fd.isDirectory()).map((fd) => fd.name);
+
+  for (dir of directoriesToImport) {
```

Comment:
Looping on the function parameter seems strange to me. I recommend creating a new variable here

Reply:
strange I thought I had corrected that 🤔 , anyways thanks for the heads up

Reply:
Looks like this is also still here

Reply:
yeah sorry, fixed now